Realm/RLMRealmConfiguration.mm

```diff
@@ -236,5 +236,16 @@ - (void)setCustomSchema:(RLMSchema *)customSchema {
     _config.schema = [_customSchema objectStoreCopy];
 }
 
+- (void)setDisableFormatUpgrade:(bool)disableFormatUpgrade
+{
+    _config.disable_format_upgrade = disableFormatUpgrade;
+    _config.cache = false;
```

Comment:
I don't think it actually needs to set `cache = false` at all, as setting this either does nothing or results in it throwing an exception, in which case the realm won't be cached anyway.